drivers/dma/dma_mcux_edma.c

```diff
@@ -532,6 +532,12 @@ static int dma_mcux_edma_reload(const struct device *dev, uint32_t channel,
 		ret = -EFAULT;
 	}
 
+	if (data->edma_handle.tcdPool == NULL) {
+		/* Not using scatter/gather mode, re-enable the hardware request */
+		EDMA_EnableChannelRequest(DEV_EDMA_HANDLE(dev, channel)->base,
```

Comment:
can you give an explanation of "why" in the commit message at least, maybe also in the comment

Reply:
added details in comment and commit message